.github/oidc/production.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "ParameterKey": "SubjectClaimFilters",
+    "ParameterValue": "repo:CMSgov/onemac:ref:refs/heads/master"
```

Comment:
master? or production?

Reply:
are you insinuating i blindly copied the val json file and didnt replace this value?!?! If you are then you are right!! :) :D

Reply:
fixed!